### DISCUSSION TITLE
SNES repro where WE is connected to VCC

### DISCUSSION THREAD

**Discussion by jon-nielsen** [sanni/cartreader#190] [Need help]
Hi!

Thanks for this project and the documentation. It's great and I've just finished building my first cart reader :-)

I've a question about a SNES repro where WE is connected to VCC. I don't see any solder jumper, but it is connected according to the multimeter. Is there a workaround so that I can flash it? And what's the significance of this connection?

The repro is setup as a 16mbit lorom with 64kbit save. At least that's what it's supposed to be to support the ROM; the game is Super Punch Out but the save function never worked. So now I just want to flash a 16mbit lorom without SRAM.

As a side note receiving this dud of a repro is what got me interested in making my own repros and tools ;-)

**Comment by sanni** [comment]
WE means write enable, it enables writing of the flashrom. If it's high, so 5V, writing will be disabled permanently.

Can you upload a picture of the repro?

**Reply by jon-nielsen** [reply]
Hello

Thanks for the reply. This makes sense - I also disable the programming pin on NES repros I make when using donors with 040 EEPROMs.

Admittedly I didn't study the schematics posted when attempting this. I've attached pictures below. As you see I've removed some components and the battery holder and attached the wire on the via. On the back side the via goes somewhere, I guess I'll try and trace it.

Have you seen this type before?

Cheers
Jon


![IMG_20210711_105111](https://user-images.githubusercontent.com/65899220/125188932-0ed6c680-e236-11eb-8940-6f927cf5997e.jpg)
![IMG_20210711_105126](https://user-images.githubusercontent.com/65899220/125188941-15653e00-e236-11eb-920c-08cca2070cd1.jpg)

**Comment by sanni** [comment]
Is there a small trace connecting those two pads? It almost looks like it on the image.

![ddddfgd585848](https://user-images.githubusercontent.com/19169494/125189583-1350ae80-e239-11eb-9143-2f4f0751ffa6.jpg)

**Reply by jon-nielsen** [reply]
Hey. Thanks for checking it out!

It was indeed connected. I've cut the trace and will now try to flash :-)

**Reply by jon-nielsen** [reply]
It works. Thanks. Cutting the trace also means having to resolder it afterwards for the game to work. I did it on another cart and I've attached pictures here if you would like to use any of them for the wiki :-)

Board:

![IMG_20210711_153847](https://user-images.githubusercontent.com/65899220/125199952-df8d7d00-e268-11eb-8b9e-103a982e91a0.jpg)

Place to cut:
![IMG_20210711_153905](https://user-images.githubusercontent.com/65899220/125199967-f207b680-e268-11eb-8332-7a03e7a8d056.jpg)

Cut:
![IMG_20210711_154357](https://user-images.githubusercontent.com/65899220/125199993-12377580-e269-11eb-939a-2fcdfdd10ed2.jpg)

Wired:
![IMG_20210711_154404](https://user-images.githubusercontent.com/65899220/125199996-1f546480-e269-11eb-881e-ab9c1f7aae65.jpg)

Restored after programming:
![IMG_20210711_155326](https://user-images.githubusercontent.com/65899220/125200006-2d09ea00-e269-11eb-88a4-cfc742b0d016.jpg)

**Comment by StanMOI** [comment]
Saves never worked?!
You need to add the battery 🔋 on your board
